modules/virt-cloning-a-datavolume.adoc

```diff
@@ -10,18 +10,15 @@
 For smart-cloning to occur, the following conditions are required.
 
 * Your storage provider must support snapshots.
-* The source and target PVCs must be defined to the same namespace.
 * The source and target PVCs must be defined to the same storage class.
-* The `VolumeSnapshotClass` object must reference the storage class defined to both the source and target PVCs.
-
-If any of these prerequisites are not met, host-assisted cloning automatically occurs when you create a data volume with a PVC source.
+* The VolumeSnapshotClass object must reference the storage class defined to both the source and target PVCs.
 
 .Procedure
 
 To initiate cloning of a data volume:
 
 . Create a YAML file for a `DataVolume` object that specifies the name of the
-new data volume, the name and namespace of the source PVC, and the size of the new data volume. This example clones a source PVC in block mode, so `volumeMode: Block` is used:
+new data volume and the name and namespace of the source PVC. In this example, because you specify the *storage* API, there is no need to specify the size of the data volume, *accessModes*, or *volumeMode*. The optimal values will be calculated for you automatically.
```

Comment:
We do need to specify the size when using *storage* API:
So remove "size of the data volume" from line 21
In the yaml you correctly specified the size in line 37 and then correctly described it in line 43

Reply:
Thanks @kgoldbla - I realized I did that yesterday and forgot to change it when I corrected the YAML. Should be good now.